src/Controller/Component/FlashComponent.php

```diff
@@ -31,13 +30,6 @@
 class FlashComponent extends Component
 {
 
-    /**
-     * The Session object instance
-     *
-     * @var \Cake\Http\Session
-     */
-    protected $_session;
```

Comment:
you can not delete `protected` member in bugfix,  for BC reasons

Reply:
We could have __...() to proxy then, right?
But yeah, people could be using the property internally right now.

Reply:
now we can add a note for this that it will remove in 4.x